packages/generator-react-server/README.md

```diff
@@ -0,0 +1,42 @@
+# generator-react-server [![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Coverage percentage][coveralls-image]][coveralls-url]
+> A react-server yeoman generator
+
+## Installation
+
+First, install [Yeoman](http://yeoman.io) and generator-react-server using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
+npm install -g generator-react-server
+```
+
+Then generate your new project:
+
+```bash
+yo react-server
+```
+
+## Getting To Know Yeoman
+
+ * Yeoman has a heart of gold.
+ * Yeoman is a person with feelings and opinions, but is very easy to work with.
+ * Yeoman can be too opinionated at times but is easily convinced not to be.
+ * Feel free to [learn more about Yeoman](http://yeoman.io/).
+
+## License
+
+Apache-2.0 © [Doug Wade](dougwade.io)
```

Comment:
Should this be Redfin at this point?  You're still listed as the `author` of the package, but Redfin is assigned copyright?